enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -126,7 +126,7 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
         LinkageCheckReport linkageReport = linkageChecker.findLinkageErrors();
         int errorCount =
             linkageReport.getJarLinkageReports().stream()
-                .mapToInt(JarLinkageReport::getCauseToSourceClassesSize)
+                .mapToInt(JarLinkageReport::getErrorCount)
```

Comment:
Strange that this is included as part of this build. Since it's already in master.

Reply:
Some weirdness in git. MIght need to rebase.

Reply:
rebase helped, thanks